### PR TITLE
implemented edit_voice function, get/edit voice settings and change in /user's response

### PR DESCRIPTION
**Pull request by krishnan-tech**
Added edit_clone function in Voice class and added some values in the response of /user.

Edit:
Added commits of get voice settings and edit voice settings functions

### CODE REVIEW

**Comment by flavioschneider** [comment]
Ah damn, I forgot to submit the review, in the meantime another PR was opened with similar features so maybe we can sync with that.

File: elevenlabs/api/voice.py [hunk 1]
```diff
@@ -129,14 +147,44 @@ def from_design(cls, voice_design: VoiceDesign):
         voice.design = voice_design
         return voice
 
+    @classmethod
+    def edit_voice(cls, voice_id: str, voice_clone: VoiceClone) -> Voice:
+        """
+        Args:
+            - voice_id - Voice ID to be used, you can use https://api.elevenlabs.io/v1/voices to list all the available voices.
```
[the remark this answers is not in the document]
Comment (krishnan-tech) [reply elevenlabs/api/voice.py:154]: I don't understand this one, what are you trying to imply? For editing voice, we will need this both parameters, what will we do by getting all voices?

[the remark this answers is not in the document]
Comment (flavioschneider) [reply elevenlabs/api/voice.py:154]: It's referred to the comment: 

> `voice_id - Voice ID to be used, you can use https://api.elevenlabs.io/v1/voices to list all the available voices.`

I'd write `you can use Voices.from_api() to list all the available voices`

[the remark this answers is not in the document]
Comment (krishnan-tech) [reply elevenlabs/api/voice.py:154]: ah yeah, sure, I'll change it :)

**Comment by flavioschneider** [comment]
I took inspiration from your PR and made a more generic `edit` function that can take a `voice_clone` object, so you should be able to use it in a similar way – either `voice.edit(...)`, or `Voice.edit_by_id(voice_id, ...)`. Some other conflicting changes were merged from the other PR. All of this is now available in `v0.2.18`. Thanks for contributing!

**Pull request closed without merging**